pkg/kernel/kernel_config-4.19.x-x86_64

```diff
@@ -3837,7 +3837,8 @@ CONFIG_EXT4_FS=y
 CONFIG_EXT4_USE_FOR_EXT2=y
 CONFIG_EXT4_FS_POSIX_ACL=y
 CONFIG_EXT4_FS_SECURITY=y
-# CONFIG_EXT4_ENCRYPTION is not set
+CONFIG_EXT4_ENCRYPTION=y
+CONFIG_EXT4_FS_ENCRYPTION=y
```

Comment:
Please make sure to update ARM config with the same settings

Reply:
Added ARM config, and updated the changes.